www/themes/Gamma/templates/frontend/profile.tpl

```diff
@@ -54,7 +58,7 @@
 		<th>Grabs Total:</th>
 		<td>{$user.grabs}</td>
 	</tr>
-	{if ($user.id==$userdata.id || $userdata.role==2) && $site->registerstatus==1}
+	{if (!$publicview || $isadmin) && $site->registerstatus==1}
```

Comment:
I'm not sure about the !$publicview. I'm reading this as if NOT public or is admin user, which doesn't make sense to me. Shouldn't it be if IS public or IS admin user?